### PR TITLE
New Example: Dogs/Cats Classifier with Keras

### PR DESCRIPTION
**Pull request by keshamin**
Example description: https://github.com/odahu/odahu-examples/tree/feat/dogs-cats-example/mlflow/keras/dogs_cats

### CODE REVIEW

File: mlflow/keras/dogs_cats/dag/dag.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                limits=ResourceList(
+                    cpu='4',
+                    memory='16Gi',
+                    gpu='1'
+                ),
+            ),
+            algorithm_source=AlgorithmSource(vcs=VCS(
+                connection='odahu-flow-examples',
+                # TODO: switch to develop
+                reference='feat/dogs-cats-example'
```
Comment (BPylypenko) [inline mlflow/keras/dogs_cats/dag/dag.py]: I suppose that it's a good time to switch.
Or are you going to make a separate PR?

**Review by BPylypenko — APPROVED** [review]

**Review by lev-titov — APPROVED** [review]
